src/main/java/com/bp3x/raidbot/commands/lfg/LFGConstants.java

```diff
@@ -37,4 +37,19 @@ private LFGConstants(){}
     public static final String DECLINED_PLAYERS = "Declined Players:";
     public static final String TENTATIVE_PLAYERS = "Tentative Players:";
     public static final String BLANK_PLAYER_LIST = "[N/A]";
+
+    public static final String LFG_HELP_START = "Example:\n`!lfg dsc 1/1/2022 9:00pm`\n";
+
+    public static final String LFG_HELP_SHORT_NAMES = "\nShort names for events: \n" +
```

Comment:
Can we build this string by iterating through all the events in the `event.json` as opposed to hard-coding it?